### DISCUSSION TITLE
List of abbreviations

### DISCUSSION THREAD

**Discussion by nsacco** [quarto-dev/quarto-cli#3502] [Q&A]
How can I create a list of abbreviations in my Quarto document for LaTeX, MS word, or HTML?

**Comment by camilogarciabotero** [comment]
See a previous discussion https://github.com/quarto-dev/quarto-cli/discussions/2483 about it.

**Reply by mcanouil** [reply]
I don't think the question was about using the abbreviation Pandoc feature, but how to create an actual list of abbreviations.

**Comment by mcanouil** [comment]
Currently Quarto does not include a way to add a list of abbreviations.
This does not mean you can't have one, but you'll have to define it yourself with the appropriate tools, especially for LaTeX and Word.

**Reply by mcanouil** [reply]
Unfortunately no, but it should not be that hard to find some on the internet.
